linters-config/html-css/.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [ ] Linter tests are not working because you have defined the wrong directory for the configuration files. Please set up the Linter correctly and make sure all tests are passing. The correct directory to install Linter :
  - .github/workflows/linters.yml

Reply:
The wrong directory has been modified to the correct path/directory.